Update egui and sdl2 versions

Manorhos opened the pull request:
I know egui 0.25 is out already, but a component I want to use in my application still depends on 0.24.1. Also, for 0.25 it seems we're gonna need scancode conversion.

I also had to bump the sdl2 version since I had dependency issues with raw-window-handle when updating the example otherwise.
You can try out the updated example at [https://github.com/Manorhos/egui-sdl2-event-example](https://github.com/Manorhos/egui-sdl2-event-example).
I can send a PR for that using the proper crate version once you've published a new version of this crate.

kaphula commented:
Much thanks for the fixes! I am not personally using this crate anymore so I have not had time to keep it up.

I created a new [branch](https://github.com/kaphula/egui-sdl2-event/tree/workspace_and_manorhos_fix) that transforms the project to use workspaces. This includes your fixes and your updated example project within this same repository. Can you check that it is alright and merge it to this branch to update this pull request. Also add credits and edit the readmes as you see fit.

I'll publish the updated library to crates.io once it is all resolved.

Manorhos commented:
Alright, did a bit of cleanup and fixed something in the readme for the example. I also switched out the `use-pkgconfig` feature in the example's TOML for `bundled` as that makes it easier to get the example up and running on Windows.
I also snuck in the removal of a `println!()` in the library crate 😉

Small disclaimer: I don't really know what I'm doing with wgpu, I just looked at the example and the egui_wgpu winit backend side-by-side and adapted the code to the new APIs as I thought best.

The example runs on my machine and the library crate does what it's supposed to in the emulator I'm using it in (using egui_glow, but with SDL2, so need something to translate the events), so from my perspective you can merge this if you're okay with the changes I made.

kaphula commented:
> Small disclaimer: I don't really know what I'm doing with wgpu, I just looked at the example and the egui_wgpu winit backend side-by-side and adapted the code to the new APIs as I thought best.

No worries, that's exactly how these things are created.

All is working nicely as far as I am concerned. I may add few cosmetic changes after the merge before publishing new version. 

Nice job, thanks!
